systers_portal/community/urls.py

```diff
@@ -2,14 +2,14 @@
 
 from community.views import (EditCommunityProfileView,
                              ViewCommunityProfileView,
-                             CommunityPageView)
+                             CommunityPageView, CommunityPageListView)
 
 urlpatterns = [
     url(r'^(?P<slug>\w+)/profile/$', ViewCommunityProfileView.as_view(),
         name='view_community_profile'),
     url(r'^(?P<slug>\w+)/profile/edit/$', EditCommunityProfileView.as_view(),
         name='edit_community_profile'),
-    url(r'^(?P<slug>\w+)/$', CommunityPageView.as_view(),
+    url(r'^(?P<slug>\w+)/$', CommunityPageListView.as_view(),
```

Comment:
I propose an approach where you could do the following. First, rename the route to `view_community_landing_page`.
Create a view somewhat called `CommunityLandingView`.

Inside the community you will perform a redirect either to the list of news view or to the community page view. Check if the community has at least one CommunityPage, then redirect to the page with the lowest order. If the community has no pages, redirect to the news list. 

Does that makes sense?